src/main/java/com/leveluptasks/controller/UserController.java

```diff
@@ -0,0 +1,55 @@
+package com.leveluptasks.controller;
+
+
+import com.leveluptasks.dto.UserDto;
+import com.leveluptasks.entity.User;
+import com.leveluptasks.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    @Autowired
```

Comment:
it will be better if u use constructor injection  instead of @Autowired because it's deprecated

Reply:
the inject with constructor is in Angular